contrib/go-chi/chi/chi_test.go

```diff
@@ -301,3 +304,79 @@ func TestIgnoreRequest(t *testing.T) {
 		assert.Equal(t, shouldSkip, len(mt.FinishedSpans()) == 0)
 	}
 }
+
+func TestAppSec(t *testing.T) {
+	appsec.Start()
+	defer appsec.Stop()
+
+	if !appsec.Enabled() {
+		t.Skip("appsec disabled")
+	}
```

Comment:
For now, this API is only used internally, but consider that this form may be confusing if and when it become public API. I mean, this is what happens:

1. AppSec gets started on line 309
2. Line 312 checks if it's enabled and skips the test, even though it was never stopped.

I understand the fact that there is an explanation for this, and that `Start` might not actually start AppSec, but nevertheless it's confusing because a functionall like `appsec.Start` which returns no error is assumed to succeed. Perhaps returning an error from `appsec.Start` (let's say `appsec.NotStarted` or something more specific which includes the reason) would be more clear.

Nothing needed in this PR, just some random feedback.

Reply:
Yes, I'm aware of this weirdness, due to how AppSec gets enabled for now: you can start/stop it, but it actually starts only if DD_APPSEC_ENABLED is true.

Reply:
Maybe appsec.Start() could return a boolean to make it simpler.
I'll take it into account next time I can.